src/Views/WhatsNewPage.xaml.cs

```diff
@@ -21,6 +22,12 @@ namespace DevHome.Views;
 
 public sealed partial class WhatsNewPage : Page

```

Comment:
I think it would be better to have all of this in the ViewModel instead. Not for this PR, though

Reply:
agree